charts/rstudio-workbench/templates/_helpers.tpl

```diff
@@ -31,14 +31,22 @@ containers:
   {{- $defaultVersion := .Values.versionOverride | default $.Chart.AppVersion }}
   image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default $defaultVersion }}"
   env:
-  {{- if .Values.enableDiagnostics }}
+  {{- if .Values.diagnostics.enabled }}
   - name: DIAGNOSTIC_DIR
-    value: "/var/log/rstudio"
+    value: "{{ .Values.diagnostics.directory }}"
   - name: DIAGNOSTIC_ONLY
     value: "true"
   - name: DIAGNOSTIC_ENABLE
     value: "true"
   {{- end }}
+  {{- if not .Values.launcher.kubernetesHealthCheck.enabled }}
+  - name: RSTUDIO_LAUNCHER_STARTUP_HEALTH_CHECK
+    value: disabled
+  {{- end }}
+  {{- if .Values.launcher.kubernetesHealthCheck.enabled }}
```

Comment:
Why not simply use `{{ - else }}` here?

Reply:
I think it was mostly in a desire to be declarative 🙈  They are not "technically" bound as if/else (i.e. right now our default on the HEALTH_CHECK header is to leave it undefined if things are enabled) so I thought it might be confusing to define them as such.